packages/main/src/ColorPicker.js

```diff
@@ -188,9 +188,10 @@ class ColorPicker extends UI5Element {
 	onBeforeRendering() {
 		// we have the color & _mainColor properties here
 		this._color = getRGBColor(this.color);
+		const tempColor = `rgba(${this._color.r}, ${this._color.g}, ${this._color.b}, 1)`;
```

Comment:
You don't need the tempColor as it is always the same as this._color

Reply:
The tempColor is the same as this._color regarding the RGB values, but not necessarily with the same alpha value,
as this._color doesn't keep track of the alpha value.